packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/outbound.ts

```diff
@@ -0,0 +1,44 @@
+// The contents of this file were taken from the AWS CDK provider framework.
```

Comment:
Like this forked file, we don't include the `defaultInvokeFunction` method from the original file. Is it because we don't need to use the function as the state machine directly invoke the amplify table handlers instead of previously going to an intermediate framework lambda?

Another question is do we have any similar `invokeLambda` method implemented somewhere (If we still need it)? If so, there is one possible bug from original CDK code [here](https://github.com/aws/aws-cdk/issues/26838) and we might take care of that later, which is the root cause of `waiter timeout error` reported by couple of customers in this [issue](https://github.com/aws-amplify/amplify-category-api/issues/2189)

Reply:
Just take a further look into the code. I think this should be fine. The CDK implementation is to invoke User lambda within its framework lambda, like invoking a nested execution in a parent execution. In your impl, I can see we have `onEventHandler` -> `processOnEvenet`, which is done within one lambda execution.

Hopefully this will address the cx waiter timeout issue. Still we need to keep an eye on it.

Reply:
That's right. The `invokeLambda` function in the CDK custom provider provider framework is for the framework lambda to invoke the "user lambda" i.e. our lambdas. This change eliminates that extra layer of lambdas, so `invokeLambda` isn't necessary.